src/native/minipal/strings.h

```diff
@@ -27,6 +27,14 @@ CHAR16_T minipal_toupper_invariant(CHAR16_T code);
  */
 CHAR16_T minipal_tolower_invariant(CHAR16_T code);
 
+/**
+ * @brief Get the length of a null-terminated UTF-16 string.
+ *
+ * @param str The null-terminated UTF-16 string.
+ * @return The length of the string.
+ */
+size_t minipal_u16_strlen(const CHAR16_T* str);
```

Comment:
I must be missing something but why not use `wcslen`?

Reply:
`wcslen` doesn't work on non-Windows as `wchar_t` on Unix platforms is 4-byte UTF-32, not 2-byte UTF-16.

Reply:
This can just forward to `wcslen` on Windows.